src/writeHelper.js

```diff
@@ -17,12 +17,11 @@ function writeLocal(agency, currentDateTime, data) {
   const version = "v1";
   const fileName = `/tmp/${agency}_${version}_${currentTimestamp}.json`
 
-  fs.writeFile(fileName, JSON.stringify(data), err => {
-    if (err) {
-      console.error(err);
-    }
-    console.log("wrote", fileName)
-  }); 
+  return writeFile(fileName, JSON.stringify(data))
```

Comment:
❓ Does this work without importing `writeFile`?

```
import { writeFile } from 'fs/promises';`
```

Reply:
Ah, sorry. Just realized this project is using CommonJS. 

```js
const { writeFile } = require('fs/promises);
```

Reply:
Yep the second way seems to work as expected, thanks for the suggestion 👍